Source/VMware.vSphereDSC/Tests/Integration/VMHostVss/VMHostVssTeaming.Integration.Tests.ps1

```diff
@@ -0,0 +1,222 @@
+$script:configurationFile = "$script:integrationTestsFolderPath\Configurations\$($script:dscResourceName)\$($script:dscResourceName)_Config.ps1"
+
+$script:configWithModifyVssTeaming = "$($script:dscResourceName)_Modify_Config"
+$script:configWithRemoveVssTeaming = "$($script:dscResourceName)_Remove_Config"
+
+$script:VssName = 'VSSDSC'
+$script:AverageBandwidth = 100000
+$script:CheckBeacon = $false
+$script:ActiveNic = @('vmnic4', 'vmnic5')
+$script:ActiveNicAlt = @('vmnic4')
+$script:StandbyNic = @()
+$script:StandbyNicAlt = @('vmnic5')
+$script:NotifySwitches = $true
+$script:Policy = 'loadbalance_srcid'
+$script:PolicyAlt = 'loadbalance_ip'
+$script:RollingOrder = $false
+$script:Present = 'Present'
+$script:Absent = 'Absent'
+
+$script:resourceWithModifyVssTeaming = @{
+    Name = $Name
+    Server = $Server
+    Ensure = $script:Present
+    VssName = $script:VssName
+    CheckBeacon = -not $script:CheckBeacon
+    ActiveNic = $script:ActiveNicAlt
+    StandbyNic = $script:StandbyNicAlt
+    NotifySwitches = -not $script:NotifySwitches
+    Policy = $script:PolicyAlt
+    RollingOrder = -not $script:RollingOrder
+    DependsOn = "[VMHostVss]vmHostVssSettings"
+}
+$script:resourceWithRemoveVssTeaming = @{
+    Name = $Name
+    Server = $Server
+    Ensure = $script:Absent
+    VssName = $script:VssName
+    CheckBeacon = $script:CheckBeacon
+    ActiveNic = $script:ActiveNic
+    StandbyNic = $script:StandbyNic
+    NotifySwitches = $script:NotifySwitches
+    Policy = $script:Policy
+    RollingOrder = $script:RollingOrder
+    DependsOn = "[VMHostVss]vmHostVssSettings"
+}
+
+. $script:configurationFile -Name $Name -Server $Server -User $User -Password $Password
+
+$script:mofFileWithModifyVssTeaming = "$script:integrationTestsFolderPath\$($script:configWithModifyVssTeaming)\"
+$script:mofFileWithRemoveVssTeaming = "$script:integrationTestsFolderPath\$($script:configWithRemoveVssTeaming)\"
+
+try {
+    Describe "$($script:dscResourceName)_Integration" {
+        Context "When using configuration $($script:configWithModifyVssTeaming)" {
+            BeforeEach {
+                # Arrange
+                $startDscConfigurationParameters = @{
+                    Path = $script:mofFileWithModifyVssTeaming
+                    ComputerName = 'localhost'
+                    Wait = $true
+                    Force = $true
+                }
+
+                # Act
+                Start-DscConfiguration @startDscConfigurationParameters
+            }
+
+            It 'Should compile and apply the MOF without throwing' {
+                # Assert
+                $startDscConfigurationParameters = @{
+                    Path = $script:mofFileWithModifyVssTeaming
+                    ComputerName = 'localhost'
+                    Wait = $true
+                    Force = $true
+                }
+                { Start-DscConfiguration @startDscConfigurationParameters } | Should -Not -Throw
```

Comment:
Change for all similar tests to:

```
 # Act && Assert
 { Start-DscConfiguration @startDscConfigurationParameters } | Should -Not -Throw
```

And remove the Assert comment above (The one above the parameters).

Reply:
Changed